### PR TITLE
Creating Helmfile for local development using OOTB images

### PR DESCRIPTION
**Pull request by uesleilima**

### CODE REVIEW

File: development/README.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+- Kubectl or K9s
+    * `brew install k9s`
+- Backbase Repository Credentials
+
+## Steps
+
+* Run: `add-backbase-helm-repo.sh`
+* Run: `add-backbase-pull-secret.sh`
+* Add: `127.0.0.1 kubernetes.docker.internal` in your `/etc/hosts` file.
+* Obtain the [employee-web-app-essentials](images/employee-web-app-essentials/README.md) image as it is not yet public
```
Comment (uesleilima) [inline development/README.md:19]: @torrespro @santam85  any updates if we are eventually going to get this guy available on repo.backbase.com ?

Comment (torrespro) [reply development/README.md:19]: as R&D is moving away from helmfile, what if we move this to a Chart of Charts? similar to https://github.com/Backbase/helm-k8s-local

Comment (torrespro) [reply development/README.md:19]: Not yet: https://backbase.atlassian.net/browse/RFF-2001

Comment (uesleilima) [reply development/README.md:19]: I have included in the readme the creation of an umbrella chart as a future improvement as we need to check how to deal with the current helmfile go template support that we use to mount the realm json in the configmap - feasible but would require extra effort

**Pull request merged**